docs/content/en/docs/getting-started/ports/_index.md

```diff
@@ -95,7 +122,7 @@ The following table displays ports that need to be accessible from the Nutanix p
 | TCP      | Inbound   | 2379        | Manager                 | Etcd API endpoint          |
 | TCP      | Inbound   | 2380        | Manager                 | Etcd API endpoint          |
 
-## Snow provider
+##### Snow provider
```

Comment:
```suggestion
### Snow provider
```
I suggested changing the heading above to ##. If you agree, then this should be ###.